sql/core/src/test/scala/org/apache/spark/sql/test/DataFrameReaderWriterSuite.scala

```diff
@@ -449,6 +449,20 @@ class DataFrameReaderWriterSuite extends QueryTest with SharedSQLContext with Be
     }
   }
 
+  test("pmod with partitionBy") {
```

Comment:
you can remove this once https://github.com/apache/spark/pull/14146 makes it in.